index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
This is good, if you wanted to refactor it a bit you could just console.log(main()) and inside main just return greet(process.argv[2]).